packages/qdrant-loader-mcp-server/src/qdrant_loader_mcp_server/mcp/intelligence_handler.py

```
@@ -30,6 +35,30 @@ def __init__(self, search_engine: SearchEngine, protocol: MCPProtocol):
         self._ttl = 300
         self._max_sessions = 500
         self._lock = asyncio.Lock()
+        self._graph_store = None
+        self._graph_store_lock = asyncio.Lock()
+
+    async def _get_graph_store(self):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_get_graph_store`

(ANN202)